src/styledobject.rs

```diff
@@ -7,39 +7,42 @@ use crossterm_utils::queue;
 
 use crate::{Attribute, Color, Colorize, ObjectStyle, ResetColor, SetAttr, SetBg, SetFg, Styler};
 
-/// Contains both the style and the content which can be styled.
+/// A styled object.
+///
+/// # Examples
+///
+/// ```rust
+/// use crossterm_style::{style, Color, Attribute};
+///
+/// let styled = style("Hello there")
+///     .with(Color::Yellow)
+///     .on(Color::Blue)
+///     .attr(Attribute::Bold);
+///
+/// println!("{}", styled);
+/// ```
 #[derive(Clone)]
 pub struct StyledObject<D: Display + Clone> {
+    /// The object style (colors, text attributes).
```

Comment:
The field 'content' on which the styled will be applied is a type that implements Display, therefore, it isn't text.
```suggestion
    /// The object style (colors, content attributes).
```

Reply:
Fixed.